core-tests/psc-package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "core-tests",
+  "set": "psc-0.11.1",
+  "source": "https://github.com/purescript/package-sets.git",
+  "depends": [
```

Comment:
Could you please add the newer core packages here too. I notice `typelevel-prelude` is missing. Thanks!